lib/pharos/root_command.rb

```diff
@@ -22,3 +22,5 @@ def self.run
     end
   end
 end
+
+Dir.glob(File.join(__dir__, '..', '..', 'non-oss', 'commands', '*.rb')).each { |non_oss_command| require non_oss_command }
```

Comment:
We probably could load `non-oss/setup.rb` or similar that handles all the commands/addons. But that kind of change is out of scope of this pr.